template/cordova/lib/PluginInfo.js

```diff
@@ -0,0 +1,109 @@
+var semver = require('semver');
+var CommonPluginInfo = require('cordova-common').PluginInfo;
+
+var MANIFESTS = {
+    'windows': {
+        '8.1.0': 'package.windows.appxmanifest',
+        '10.0.0': 'package.windows10.appxmanifest'
+    },
+    'phone': {
+        '8.1.0': 'package.phone.appxmanifest',
+        '10.0.0': 'package.windows10.appxmanifest'
+    },
+    'all': {
+        '8.1.0': ['package.windows.appxmanifest', 'package.phone.appxmanifest'],
+        '10.0.0': 'package.windows10.appxmanifest'
+    }
+};
+
+/*
+A class for holidng the information currently stored in plugin.xml
+It's inherited from cordova-common's PluginInfo class
+In addition it overrides getConfigFiles method to use windows-specific logic
+ */
+function PluginInfo(dirname) {
+    CommonPluginInfo.apply(this, arguments);
+    var parentGetConfigFiles = this.getConfigFiles;
+    this.getConfigFiles = function(platform) {
+        var changes = parentGetConfigFiles(platform);
+        var editChanges = this.getEditConfigs(platform);
+        if (editChanges && editChanges.length !== 0) {
+            changes.push(editChanges);
+        }
+        // Demux 'package.appxmanifest' into relevant platform-specific appx manifests.
+        // Only spend the cycles if there are version-specific plugin settings
+        if (changes.some(function(change) {
+                    return change.target === 'package.appxmanifest';
```

Comment:
Changes w/ target 'package.appxmanifest' and w/o versions and deviceTarget used to demux into manifests in   remove_plugin_changes and munge_helper methods. This logic is moved in PluginInfo.js (see also https://github.com/apache/cordova-windows/pull/207/commits/b101deda784bfdb3ad45c2f86b830c6cc356a72b?diff=unified#diff-93a26ea6e269125a5b151c4176a3b42bR54).
Once windows-specific logic is completed there aren't any changes with 'package.appxmanifest' target.